src/main/java/uk/gov/hmcts/darts/common/repository/TranscriptionRepository.java

```diff
@@ -152,16 +152,14 @@ List<TranscriptionSearchResult> searchMigratedTranscriptionsFilteringOn(
         SELECT distinct t
         FROM TranscriptionEntity t
         WHERE t.requestor = str(:userId)
-        AND ((:onOrAfterCreatedDate IS NULL) OR 
-        (:onOrAfterCreatedDate IS NOT NULL AND t.createdDateTime >= :onOrAfterCreatedDate))
+        AND ((cast(:onOrAfterCreatedDate as TIMESTAMP)) IS NULL OR t.createdDateTime >= :onOrAfterCreatedDate)
```

Comment:
I wonder how come the existing integration test didn't spot this issue

Reply:
Writing a message to the channel as we speak